tower-http/src/cors.rs

```diff
@@ -678,7 +678,7 @@ where
                         return ResponseFuture {
                             inner: Kind::Error(Some(
                                 Response::builder()
-                                    .status(StatusCode::METHOD_NOT_ALLOWED)
+                                    .status(StatusCode::OK)
```

Comment:
The two branches here acn now be combined by changing the above `if` to a match guard.